publish/publish.proj

```diff
@@ -49,6 +49,26 @@
                    Items="@(ItemsToPublish)" 
                    Overwrite="false" />
   </Target>
+  
+  <Target Name="UploadChecksumsToAzure"
```

Comment:
The SDK also updates the "channel" during finalize build so we can always have a correct "latest" link.  However, given the current state of things, this would probably be easiest done after @mellinoe's change.  So I'm fine with the first check-in for this feature to not account for "finalize build".  But once #2385 is merged, we should update the checksums to be 'finalized' as well.